diff -Naur a/lib/SignOn/doc/doc.pri b/lib/SignOn/doc/doc.pri
--- a/lib/SignOn/doc/doc.pri	2013-04-10 12:29:05.000000000 +0200
+++ b/lib/SignOn/doc/doc.pri	2013-04-17 17:16:50.417006316 +0200
@@ -58,7 +58,11 @@
 #-----------------------------------------------------------------------------
 # Installation target setup for documentation
 #-----------------------------------------------------------------------------
-documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt
+greaterThan(QT_MAJOR_VERSION, 4) {
+    documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt5
+} else {
+    documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt
+}
 for( folder, DOC_FOLDERS ) {
     documentation.files += $${folder}
 }
