From 00a7860b4c16c1f63f62dfba874ecf57ed407e13 Mon Sep 17 00:00:00 2001
From: Robin Burchell <robin+git@viroteck.net>
Date: Thu, 11 Jul 2013 08:53:06 +0000
Subject: [PATCH 2/9] fix documentation path

---
 lib/SignOn/doc/doc.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/SignOn/doc/doc.pri b/lib/SignOn/doc/doc.pri
index 46f1148..3f2a96b 100644
--- a/lib/SignOn/doc/doc.pri
+++ b/lib/SignOn/doc/doc.pri
@@ -58,7 +58,7 @@ include( ../../../common-installs-config.pri )
 #-----------------------------------------------------------------------------
 # Installation target setup for documentation
 #-----------------------------------------------------------------------------
-documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt
+documentation.path = $${INSTALL_PREFIX}/share/doc/libsignon-qt5
 for( folder, DOC_FOLDERS ) {
     documentation.files += $${folder}
 }
-- 
2.17.1

